backend/infrahub/config.py

```diff
@@ -378,6 +380,23 @@ class WorkflowSettings(BaseSettings):
     worker_polling_interval: int = Field(
         default=2, ge=1, le=30, description="Specify how often the worker should poll the server for tasks (sec)"
     )
+    worker_type: str = Field(
+        default="process", description="Type of worker to run for the execution of user specific tasks"
+    )
+    work_pools: list[WorkerPoolDefinition] = Field(
+        default=[
+            WorkerPoolDefinition(
+                name="infrahub-worker",
+                workflow_type=WorkflowType.INTERNAL | WorkflowType.CORE | WorkflowType.USER,
```

Comment:
How would the configuration looks like when using environment variables if we want to configure multiple work_pools ?

Reply:
Also, not sure if having that as part of the configuration will work well  .. ideally the worker pools should be pre-defined by the community and the enterprise version